projects/helper/chain/cosmos.js

```diff
@@ -203,6 +203,53 @@ async function sumTokens({ balances = {}, owners = [], chain, owner }) {
   return transformBalances(chain, balances);
 }
 
+async function tvlFromLockedGammTokens(contractAddresses, chain) {
```

Comment:
I know it is still in draft mode, but if this code is linked with only one project/adapter, I dont think it should be common helpers file